src/porepy/numerics/fv/mpfa.py

```diff
@@ -598,18 +620,25 @@ def _mpfa_local(g, k, bnd, eta=None, inverter="numba", apertures=None):
     bound_exclusion = fvutils.ExcludeBoundaries(subcell_topology, bnd, g.dim)
 
     # No flux conditions for Dirichlet boundary faces
```

Comment:
Does this also mean we assign Robin conditions on the rows that represent Neumann conditions?

Update comment, please.